### DISCUSSION TITLE
Update existing dependency dashboard when `mode=silent`

### DISCUSSION THREAD

**Discussion by sisp** [renovatebot/renovate#29882] [Suggest an Idea]
### Tell us more.

I'm trying to use [`mode=silent`](https://docs.renovatebot.com/configuration-options/#mode) for triggering a rebase-only Renovate run on GitLab. It works as expected except that when the dependency dashboard is enabled and I tick a checkbox there to request a rebase of a MR, Renovate won't reset the checkbox afterwards. As of #29866, the documentation clarifies that Renovate will neither create new issues nor update existing issues (such as the dependency dashboard) when `mode=silent` is used. However, Renovate does honor a ticked checkbox in an _existing_ dependency dashboard, so not updating the _existing_ dependency dashboard (i.e., unticking the checkbox again) seems a bit inconsistent.

Hence, I suggest to extend Renovate to update an _existing_ dependency dashboard like with `mode=full` while retaining any other current behavior of `mode=silent` as is. If you agree to this idea, I'd be happy to submit a PR.

WDYT?

**Comment by RahulGautamSingh** [comment]
`mode-silent` which means issues will not be updated. So, it does not make sense to rebase PRs by interacting with issues when `mode=silent`. The approach seems wrong to me, rather than the behaviour being inconsistent.

If you want to rebase PRs through UI interaction when `mode=silent`, you can try adding `rebase` label to those prs. [^1]

[^1]: https://docs.renovatebot.com/updating-rebasing/#manual-rebasing
